include/deal.II/fe/fe.h

```diff
@@ -229,15 +260,23 @@ namespace hp
  * the refined lines, since these must be mapped one-to-one to the appropriate
  * dofs of the vertices of the unrefined line.
  *
- * It should be noted that it is not possible to distribute a constrained
- * degree of freedom to other degrees of freedom which are themselves
- * constrained. Only one level of indirection is allowed. It is not known at
- * the time of this writing whether this is a constraint itself.
+ * Through this construction, the degrees of freedom on the child faces are
+ * constrained to the degrees of freedom on the parent face. The information
+ * so provided is typically consumed by the
+ * DoFTools::make_hanging_node_constraints() function.
  *
+ * @note The hanging node constraints described by these matrices is only
+ *   relevant to the case where the same finite element space is used on
+ *   neighboring (but differently refined) cells. The case that the finite
+ *   element spaces on different sides of a face are different, i.e.,
+ *   the $hp$ case (see @ref hp "hp finite element support") is handled
+ *   by separate functions. See the FiniteElement::get_face_interpolation_matrix()
+ *   and FiniteElement::get_subface_interpolation_matrix() functions.
```

Comment:
Valid point. Last I looked this up, there were a couple of the more difficult elements (Nedelec?) that still used this way. It looked not straightforward what one would have to do here. I must admit that I'm not up for it to fix this anytime soon.

I'll open an issue though to remind us of it.